educators-corner iRISE-FORRT collab

rachelHey opened the pull request:
As requested by Flavio, I wrote a short blog post for the FORRT educators corner on the iRISE - FORRT collaboration that led to a paper in Royal Society Open Science that is now available online.

flavioazevedo commented on content/educators-corner/022-repro-metrics-forrt-irise/_index.md:
```diff
@@ -0,0 +1,78 @@
+### EDI considerations 
+
+We also explored how often dimensions related to equity, diversity and inclusion (EDI) were explicitly mentioned or addressed in the included literature. Unsurprisingly, only a few papers mentioned EDI dimensions, which highlights the need to make these considerations more explicit in future replication efforts. Within iRISE, EDI and research culture are not an afterthought; they are embedded by  ...
+
```
> Dr. Flávio Azevedo, FORRT's Director, sits at iRise's *Scientific and Ethical Advisory Board* where he hopes to help iRise's SEAB to monitor ethical issues that might require attention over the course of the project, but also monitor the progress of our work, assess its scientific quality, and provide feedback to consortium members.

rachelHey commented:
Thank you @richarddushime and @flavioazevedo - I renamed the file as requested, I added the text snippet Flavio suggested, put the names of the FORRT members who joined  the effort and added a screenshot and a bit more context for the website with the interactive version of the Table. 

Let me know if it is fine now. 

Best, 
Rachel
